docs/genai/deployment/deploy-serving-graph-as-job.ipynb

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        inputs = {"data": input_path}
</pre><p><code>input_path</code> is undefined. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8900/discussion/#comment-2521623192'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>